### PR TITLE
Add new function `src/app/core/utils/geo::createFeatureFromFeatureObject({ id, feature={} })` (copy and paste of feature attributes)

### LINKED ISSUE #9: Copy and paste from other layer
In the selection / confirmation session of the feature to be pasted, I would see more information.

immagine

For example, the list of all (or of the first three) fields of the associated record

Such as seen when merging multiple features (keeping the drop and the selection checkbox)

immagine

### PR DESCRIPTION
**Pull request by volterra79**
Closes: https://github.com/g3w-suite/g3w-client-plugin-editing/issues/9

### CODE REVIEW

File: src/app/core/layers/geolayermixin.js [hunk 1]
```diff
@@ -64,17 +64,11 @@ proto.getOlSelectionFeature = function(id){
   return this.olSelectionFeatures[id];
 };
 
-proto.updateOlSelectionFeature = function({id, geometry}={}){
+proto.updateOlSelectionFeature = function({id, feature}={}){
```
Comment (Raruto) [inline src/app/core/layers/geolayermixin.js:67]: @volterra79 beware that this edit may not be backwards compatible, for example:

https://github.com/g3w-suite/g3w-client/blob/42781b4c12903419a07762563181652484ccc7b0/src/app/core/layers/tablelayer.js#L139

Is it also used by other plugins?

File: src/app/core/layers/geolayermixin.js [hunk 2]
```diff
@@ -103,9 +97,9 @@ proto.getOlSelectionFeatures = function(){
   return this.olSelectionFeatures;
 };
 
-proto.addOlSelectionFeature = function({id, geometry}={}){
+proto.addOlSelectionFeature = function({id, feature}={}){
```
Comment (Raruto) [inline src/app/core/layers/geolayermixin.js:100]: @volterra79 the same goes for this other function, for example:

https://github.com/g3w-suite/g3w-client/blob/6e5b47a672ed6bd62cbc6f92f515c3effc1639c5/src/app/gui/table/tableservice.js#L160-L163

https://github.com/g3w-suite/g3w-client/blob/6e5b47a672ed6bd62cbc6f92f515c3effc1639c5/src/app/gui/table/tableservice.js#L253-L256

https://github.com/g3w-suite/g3w-client/blob/6e5b47a672ed6bd62cbc6f92f515c3effc1639c5/src/app/gui/table/tableservice.js#L160-L163

File: src/app/core/layers/geolayermixin.js (hunk 1 above)
Comment (volterra79) [reply src/app/core/layers/geolayermixin.js:67]: It use just internally. Not plugin use it

File: src/app/core/layers/geolayermixin.js (hunk 2 above)
Comment (volterra79) [reply src/app/core/layers/geolayermixin.js:100]: It used by queryresultservice.js when click on selection tool

**Pull request merged**